spinalcordtoolbox/deepseg/models.py

```diff
@@ -293,11 +293,12 @@
          'url': 'https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/',
          'models': ['model_seg_sc_contrast_agnostic_softseg_monai']},
     'seg_sc_lesion_t2w_sci':
-        {'description': 'Traumatic SCI spinal cord/lesion seg for T2w contrast',
```

Comment:
Note that this description is displayed when running `sct_deepseg -h`, so it might be worth keeping it short (to avoid wrapping in the table):

![image](https://github.com/user-attachments/assets/87846b29-302e-4249-b936-f1574f4d2a8a)

Reply:
Done in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/4583/commits/5cdc2e6f25ed78ce5cf40e88173413fc22fe014f, I believe it should be fine now?